src/main/java/com/diffblue/javademo/tictactoe/TicTacToeGame.java

```diff
@@ -0,0 +1,101 @@
+package com.diffblue.javademo.tictactoe;
+
+
+public class TicTacToeGame {
+
+  private Board board = new Board(3);
+
+
+  /**
+   * Check move of players.
+   */
+  public boolean isValidMove() {
+    int diff;
+    int countO = 0;
+    int countX = 0;
+    for (int row = 0; row < board.getSize(); row++) {
+      for (int col = 0; col < board.getSize(); col++) {
+        if (board.getPlayer(row, col) == Player.O) {
+          ++countO;
+        }
+        if (board.getPlayer(row, col) == Player.X) {
+          ++countX;
+        }
+      }
+    }
+    // Fail if one player has made too many moves compared to the other
+    if (countO > countX) {
+      int tmp = countO;
+      countO = countX;
+      countX = tmp;
+    }
+    diff = Math.abs(countO - countX);
```

Comment:
Is there a good reason for using Math.abs here?

Reply:
Not really, but since coverage team modelled `Math.abs`, wanted to have somewhere to demonstrate.